src/Survey/ItechArt.Survey.WebApp/Program.cs

```diff
@@ -18,6 +20,10 @@ public static void Main(string[] args)
 
     public static IHostBuilder CreateHostBuilder(string[] args)
         => Host.CreateDefaultBuilder(args)
+            .UseSerilog((context, configuration) => configuration.ReadFrom.Configuration(context.Configuration)
+            .WriteTo.File(
+                path: $"{AppDomain.CurrentDomain.BaseDirectory}{context.Configuration.GetValue<string>("Serilog:WriteToFile:path")}",
```

Comment:
avoid using default named parameters
if necessary - you can create a separate variable